src/notation/internal/notationplayback.cpp

```diff
@@ -299,7 +299,7 @@ const Tempo& NotationPlayback::tempo(tick_t tick) const
         return empty;
     }
 
-    m_currentTempo.valueBpm = static_cast<int>(score()->tempomap()->tempo(tick).toBPM().val);
+    m_currentTempo.valueBpm = std::round(score()->tempomap()->tempo(tick).toBPM().val);
```

Comment:
but `std::round()` does return a `double`, so that cast to `int` would still be needed (as `valueBpm` is an `int`), for the compiler not to warn, as far as I can tell?

Reply:
Yeah that's correct. I realized that after creating the PR and forgot to push the changes. Thanks! :)

Reply:
Strange enough neither MSVC nor MinGW do warn about it?!

Reply:
Yeah, that's probably correct. I figured it out because of a hunch and not because of a warning